storybook/public/layouts/search/test/nuxeo-test-search-results.html

```diff
@@ -0,0 +1,75 @@
+<!-- XXX: stub for the <nuxeo-results> element because it's still part of the nuxeo/nuxeo-web-ui repo. -->
```

Comment:
nit: do we have some tickets we can reference here?

Reply:
I just created this one: ELEMENTS-1321: Review layout loading mechanism on nuxeo-search-results-layout
I'll reference it.